analyses/tests/test_psp_fitting.py

```diff
@@ -1,3 +1,8 @@
+'''tests the psp_fits function.  If tests fail can use check_psp_fitting() to 
```

Comment:
Does this file test anything that is not already tested in neuroanalysis?

Reply:
It tests the old code that is in multipatch which has defaults baked in.